packages/webpack/src/utils/style-loader.js

```diff
@@ -11,6 +11,7 @@ export default class StyleLoader {
     this.perfLoader = perfLoader
     this.rootDir = options.rootDir
     this.loaders = {
+      ...options.build.loaders,
```

Comment:
This will lead to all loaders which are not related to style like `vue-loader`, `file-loader`, `ts-loader...` are wrapped on style-loader.

Reply:
I suggest only add what's needed like `loaders.vueStyle` in it.

Reply:
Sounds much more efficient and safe 👍 

I just pushed the change!
@clarkdo